docs/user/announcements.rst

```diff
@@ -2,22 +2,63 @@
 Announcement strings
 ####################
 
-The ``announcements.yml`` file is where all of MobArena's announcements
-and messages are stored. You can freely edit this file however you see
-fit, and color codes are supported. **To add color to a message**, use
-the ``&``-character followed by a valid color code. You can find a list
-of valid codes `right
-here <http://minecraft.gamepedia.com/Formatting_codes#Color_codes>`__,
-and you can find examples in the default file.
-
-Note that some of the announcements take a variable, represented in the
-message by a ``%``-character. You can leave this character out if you
-don't want the variable in the message. It is not possible to add
-variables to announcements that don't take them by default, and
-announcements that take one variable cannot take more than that one
-variable.
-
-If you **don't want to see a specific announcement**, you have to set
-its value to ``''`` (note: two single-quotes, not one double-quote).
-This will cause MobArena to ignore the announcement. Note that simply
-removing the node will not work, as MobArena forcefully adds it back in.
+MobArena supports custom strings for announcement and messages. This lets you
+create custom messages unique to your server based on certain actions, like a
+game ending. The ``announcements.yml`` file stores MobArena announcement and
+message strings. [#]_
+
+.. [#] Default strings are found in the `Msg.java`_ class
+
+
+*************
+Color support
+*************
+
+Color codes are supported. **To add color to a message**, use an ampersand
+(``&``) followed by a valid color code. `Color codes`_ are available in the
+Gamepedia Minecraft wiki. Examples are in the default file.
+
+
+*********
+Variables
+*********
+
+Some announcements use a variable. In the string, variables are represented by a
+percent sign (``%``). Remove it if you do not want a variable in the message.
+
+There are some limitations to variables:
+
+- Not possible to add variables to announcements that don't take them by default
+- Announcements that take one variable cannot take more than that variable
+
+
+***********************
+Disable an announcement
+***********************
+
+If you want to **disable a specific announcement**, set its value to two single
+quotes (``''``). MobArena ignores announcements set to an empty value. To
+disable one, you must override it. Deleting the option does not disable an
+announcement since MobArena adds them in by default.
+
+
+********
+Examples
+********
+
+Three examples below show all of the features explained above.
```

Comment:
The examples don't seem to be shown here on github in the preview. Is that a bug in the markdown or a "missing feature" of github? I've seen code blocks in other places o.O

Reply:
@garbagemule The code block doesn't render on GitHub, but it will on ReadTheDocs. This type of code block is something that is either unique to Sphinx or ReadTheDocs, I can't remember which. If you run `make clean html` or the `Make.bat` file on Windows, it should build and render the docs in HTML.

tl;dr: It will look correct in the RTD site. :)

Reply:
Roger :)